daemon/cmd/daemon.go

```diff
@@ -803,6 +820,21 @@ func (d *Daemon) TriggerReloadWithoutCompile(reason string) (*sync.WaitGroup, er
 	return d.endpointManager.RegenerateAllEndpoints(regenRequest), nil
 }
 
+func (d *Daemon) datapathRegen(reasons []string) {
+	log.Debugf("Regenerating all endpoints with datapath rewrite")
+	reason := strings.Join(reasons, ", ")
+
+	regenerationMetadata := &regeneration.ExternalRegenerationMetadata{
+		Reason:            reason,
+		RegenerationLevel: regeneration.RegenerateWithDatapathRewrite,
+	}
+	d.endpointManager.RegenerateAllEndpoints(regenerationMetadata)
```

Comment:
I believe regenerating all endpoints will already log a msg, do we need to log this again?